<?xml version="1.0"?>
<robot name="test_gripper">
  
  <include filename="$(find pr2_defs)/calibration/default_cal.xml" />
  <include filename="$(find pr2_defs)/defs/gripper/gripper_defs.urdf.xacro" />
  <include filename="$(find pr2_defs)/defs/materials.urdf.xacro" />
  
  <pr2_gripper side="r" parent="base_link" screw_reduction="${4.0/1000.0}"
               gear_ratio="${29.16*22.0/16.0*0.9}" >
    <origin xyz="0 0 0" rpy="0 0 0" />
  </pr2_gripper>
  
  <!-- Solid Base for visualizer -->
  <joint name="base_joint" type="fixed">
    <parent link="world" />
    <origin xyz="0 0 0" rpy="0 0 0" /> 
    <child link="base_link" />
  </joint>

  <link name="base_link">
    <inertial>
      <mass value="1000" />
      <origin xyz=" 0 0 0 " /> 
      <inertia ixx="1000" ixy="0" ixz="0" iyy="1000" iyz="0" izz="1000" />
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0 " /> 
      <geometry name="pr2_base_mesh_file">
        <mesh scale="20 20 0.01" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0.0 0.0 0.0 " />
      <geometry name="base_collision_geom">
        <box size="1 1 1" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="base_link" >
    <material>Gazebo/White</material>
  </gazebo>



</robot>
